packages/flutter/test/widgets/draggable_test.dart

```diff
@@ -3414,6 +3442,34 @@ void main() {
     await tester.pumpAndSettle();
   });
 
+  testWidgets('Drag and drop - feedback matches pointer in scaled MaterialApp', (WidgetTester tester) async {
+    await tester.pumpWidget(Transform.scale(
+      scale: 0.5,
```

Comment:
Nit: Is it worth it to add another test with rotation?

Reply:
Done ✅